Source/Blazorise/wwwroot/datePicker.js

```diff
@@ -39,8 +39,8 @@ export function initialize(element, elementId, options) {
     const defaultOptions = {
         enableTime: options.inputMode === 1,
         dateFormat: options.inputMode === 1 ? 'Y-m-d H:i' : 'Y-m-d',
-        allowInput: !(options.disableMobile || false),
-        altInput: !(options.disableMobile || false),
+        allowInput: !(isMobile() && (options.disableMobile || false)),
+        altInput: !(isMobile() && (options.disableMobile || false)),
```

Comment:
My mind can't think today :D Is this right? Does it handle every case?
Might be a better read if you just bring the isMobile() out.
I.E 
!IsMobile() || (isMobile()  && !options.disableMobile)

But I think it's those pipes that are killing me, again is it a coalesce operation?

Reply:
The logic is actually broken on flatpickr side. They don't detect the mobile device properly. So I was trying to come up with a workaround, even pushed a PR on their repo, but it still seems not to detect all of the mobile devices.